oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -28,7 +28,7 @@ OEP-55: Project Maintainers
    * - Resolution
      - <links to any discussions where the final status was decided>
    * - References
-     - <links to any other relevant discussions or relevant related materials>
+     - `Follow Up Work <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3426844690/Maintainership+Pilot>`
```

Comment:
```suggestion
     - `Follow-up Work <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3426844690/Maintainership+Pilot>`_
```

* s/Follow Up/Follow-up
* trailing underscore is needed to turn this into an RST link